lesson_2/src/main/java/com/sakovich/dao/UserDao.java

```
@@ -0,0 +1,60 @@
+package com.sakovich.dao;
+
+import com.sakovich.bd.DataSourceUtil;
+
+
+import java.sql.*;
+
+public class UserDao {
+
+    public static int findByLogin(String login, String password) {
+        try {
+            Connection connection = DataSourceUtil.INSTANCE.getConnection();
+
+            PreparedStatement statement = connection.prepareStatement("select * from users where login=?");
+            statement.setString(1, login);
+
+            ResultSet resultSet = statement.executeQuery();
+            //если в бд нет такого логина, возвращаем false
+            if (!resultSet.next()) {
+                DataSourceUtil.INSTANCE.returnConnection(connection);
+                return 0;
```

better to throw an exception if you didn't found expected entry